src/types/interfaces.ts

```diff
@@ -137,6 +137,7 @@ export interface ITemplate {
   assigneeType: string
   workflowStateId: string
   createdBy: string
+  createdAt: Date
```

Comment:
We need support for updatedAt since templates are sorted by most recently modified.